### ISSUE TITLE
Expand Drawer outward instead of inward

### ISSUE THREAD

**Issue by jameskyle** [alloy/macvim#48]
It would be a nice feature to expand the drawer outward instead of inward so that the drawer does not consume editor space.

Cheers and keep up the good work, mate!

**Comment by tobia** [comment]
You mean enlarging the window when the drawer is shown and reducing it when it is hidden? If so, I agree, it would be useful.

**Comment by rdeits** [comment]
Agreed -- It's easy to write a vimscript function to get around this but it shouldn't be necessary.

**Comment by indirect** [comment]
Anyone want to take pity on me and paste the vimscript function to get around this?
